python/ray/tune/suggest/bohb.py

```diff
@@ -0,0 +1,77 @@
+"""BOHB (Bayesian Optimization with HyperBand)"""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from hpbandster import BOHB
+from ray.tune.suggest import SuggestionAlgorithm
+
+
+class JobWrapper():
+    """Dummy class"""
+
+    def __init__(self, loss, budget, config):
+        self.result = {"loss": loss}
+        self.kwargs = {"budget": budget, "config": config.copy()}
+        self.exception = None
+
+class TuneBOHB(SuggestionAlgorithm):
+    """Tune Suggestion Algorithm for BOHB code"""
+
+    def __init__(self,
+                 space,
+                 max_concurrent=8,
+                 metric="neg_mean_loss",
```

Comment:
can you also add the `mode` parameter (which says, "max" or "min") and adjust the result accordingly? You can keep `JobWrapper.result` as `loss`, but the input to that should be negated if "mode == max".

Reply:
Are you sure the input should be negated if the mode is max? It looks like when it is used in other suggestion algorithms it is negated when mode is min.